src/main/java/com/google/sps/data/User.java

```diff
@@ -22,4 +27,12 @@ public User(String id, boolean isMaintainer) {
     this.id = id;
     this.isMaintainer = isMaintainer;
   }
+
+  public Filter getUserFilter() {
```

Comment:
This is a userFilter against a the user table right? 

getUserFilterForUserTable?

Reply:
I was thinking this could operate for a single user that is requesting their orgs to be listed, not the table of all users